app/views/layouts/_messages.html.haml

```diff
@@ -0,0 +1,7 @@
+.container
+  .row
+    .col-xs-10.col-xs-offset-1
+      - flash.each do |name, msg|
+        %div{:class => "alert alert-#{name}"}
+          %a{:data-dismiss => "alert", :href => "#", :class => "close"} &#215
+          = content_tag :div,  msg, :id => "flash_#{name}" if msg.is_a?(String)
```

Comment:
Consider using current ruby syntax instead of hash rockets. E.g:
```
%div{ class: "alert alert-#{name}" }
```
Also please remember to add empty space after `{` and before `}`. Here and in all other files.